### DISCUSSION TITLE
Is DotNet's NativeAoT supported?

### DISCUSSION THREAD

**Discussion by ThaDaVos** [EWSoftware/SHFB#1073] [Q&A]
I haven't tried it yet - but was curious if DotNet's NativeAoT is supported or not - I know one can generate XML doc with NativeAoT, but I thought I read somewhere that Reflection was used too

**Comment by EWSoftware** [comment]
Native code assemblies can't be parsed.  They have to be IL.  If you can generate them without AOT compilation they should work.
